services/proxy/pkg/middleware/create_home.go

```diff
@@ -65,6 +46,28 @@ func (m createHome) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	//ctx := tokenpkg.ContextSetToken(r.Context(), token)
```

Comment:
leftover?

Reply:
Was there before :stuck_out_tongue: But fixed it anyways...